homeassistant/components/rointe/config_flow.py

```diff
@@ -0,0 +1,141 @@
+"""Config flow for Rointe Heaters integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from rointesdk.rointe_api import RointeAPI
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import (
+    CONF_INSTALLATION,
+    CONF_LOCAL_ID,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    DOMAIN,
+    LOGGER,
+)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle the config flow for Rointe Heaters."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Config flow init."""
+        self.step_user_data: dict[str, Any] | None = None
+        self.step_user_local_id: str | None = None
+        self.step_user_installations: dict[str, Any] | None = None
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Step: User credentials validation."""
+
+        errors: dict[str, str] = {}
```

Comment:
Not point in defining these before we check whether the user has provided input.